clients/cobol-lsp-vscode-extension/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the COBOL Language Support extension are documented in th
 ## [2.1.1](https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/compare/2.1.0...2.1.1) (2024-01-30)
 #### Fixed
 - Miscellaneous bug fixes
+- Fix new tab for cobol code in Endevor 4 explorer
```

Comment:
Explorer for Endevor

Reply:
what was exactly the issue? smth like: Fixed broken links in Problems tab in Explorer for Endevor?